Description: avoid too-new-in-Debian crate opendal
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2025-08-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,10 +60,8 @@
 number_prefix = "0.4"
 object = "0.36"
 once_cell = "1.19"
-opendal = { version = "0.52.0", optional = true, default-features = false }
 rand = "0.8.4"
 regex = "1.10.3"
-reqsign = { version = "0.16.0", optional = true }
 reqwest = { version = "0.12", features = [
   "json",
   "blocking",
@@ -118,25 +116,9 @@
 [features]
 all = [
   "dist-client",
-  "redis",
-  "s3",
-  "memcached",
-  "gcs",
-  "azure",
-  "gha",
-  "webdav",
-  "oss",
 ]
-azure = ["opendal/services-azblob", "reqsign", "reqwest"]
 default = ["all"]
-gcs = ["opendal/services-gcs", "reqsign", "url", "reqwest"]
-gha = ["opendal/services-ghac", "reqwest"]
-memcached = ["opendal/services-memcached"]
 native-zlib = []
-oss = ["opendal/services-oss", "reqsign", "reqwest"]
-redis = ["url", "opendal/services-redis"]
-s3 = ["opendal/services-s3", "reqsign", "reqwest"]
-webdav = ["opendal/services-webdav", "reqwest"]
 # Enable features that will build a vendored version of openssl and
 # statically linked with it, instead of linking against the system-wide openssl
 # dynamically or statically.
